Decred Wallet/Features/Navigation Menu/TabMenu.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  TabMenu.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+import Signals
+
+class TabMenu: UIView {    
```

Comment:
For long, I wondered what `TabMenu` represented. `TabMenuBar` is a better name or `TabNavigationMenu`

Reply:
renamed and references resolved in 37e2453